TheFoodieNetwork/app/src/main/java/com/fbu/thefoodienetwork/keys/ParcelKeys.java

```diff
@@ -0,0 +1,6 @@
+package com.fbu.thefoodienetwork.keys;
+
+public interface ParcelKeys {
+    String selectedRestaurant = "selectedRestaurant";
+    String selectedUser = "selectedUser";
```

Comment:
Nice!!!  Only thing that I would ask, here, is that, since these are constants, they should be upper cased with underscores as delimiters: `selectedRestaurant` becomes `SELECTED_RESTAURANT`